package.json

```diff
@@ -22,7 +22,8 @@
     "react-icons": "^3.10.0",
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.4.1",
-    "typescript": "~3.7.2"
+    "typescript": "~3.7.2",
+    "yarn": "^1.22.4"
```

Comment:
We don't need to install yarn in project.
Please remove it and just install it I your global environment.